src/components/header/header.styles.js

```diff
@@ -2,26 +2,30 @@ import styled from 'styled-components';
 import { NavLink } from 'react-router-dom';
 
 export const NavContainer = styled.nav`
-  height: 100px;
-  margin-left: 80px;
-  margin-right: 80px;
```

Comment:
In my opinion, this was a nicer solution compared to setting the margins on the child components. This way you have the margins in one place which makes it easier to read and understand